geniza/corpus/management/commands/merge_duplicate_joins.py

```diff
@@ -0,0 +1,45 @@
+"""
+To be run immediately after import_data. This script operates under the assumption 
+    that a user has not taken advantage of the unique features offered by the 
+    admin UI, but conforms to the expected format provided by the metadata spreadsheet.
+"""
+
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Collapse PGPIDs that share the same joins"""
+
+    def get_documents_with_multiple_joins(self):
+        pass
```

Comment:
Here's the way I thought of for identifying candidates:  annotate document queryset with fragment count (something like `annotate(fragment_count=models.Count("fragments")`, filter to documents with more than one fragment, and then get a list of ids (`.value_list('pk', flat=True)`)